_includes/footer.html

```diff
@@ -16,21 +16,22 @@
     </ul>
     <ul>
       <li><h5>Community</h5></li>
-      <li><a href="{{ site.baseurl }}/community/index.html#mailing_lists">Mailing lists</a></li>
-      <li><a href="{{ site.baseurl }}/community/index.html#irc">Social Media</a></li>
+      <li><a href="{{ site.baseurl }}/community/">Community</a></li>
+      <li><a href="{{ site.baseurl }}/community/index.html#mailing-lists">Mailing Lists</a></li>
+      <li><a href="{{ site.baseurl }}/community/index.html#chat-rooms">Chat Rooms &amp; More</a></li>
+      <li><a href="{{ site.baseurl }}/community/index.html#community-libraries-and-tools">Libraries and Tools</a></li>
     </ul>
     <ul>
       <li><h5>Contribute</h5></li>
-      <li><a href="{{ site.baseurl }}/contribute/bug-reporting-guide.html">Report an Issue</a></li>
       <li><a href="{{ site.baseurl }}/contribute">How to Help</a></li>
-      <li><a href="{{ site.baseurl }}/contribute/hacker-guide.html">Contributor's Guide</a></li>
-      <li><a href="{{ site.baseurl }}/contribute/scala-fame.html">Contributor's Hall of Fame</a></li>
+      <li><a href="{{ site.baseurl }}/contribute/bug-reporting-guide.html">Report an Issue</a></li>
+      <li><a href="{{ site.baseurl }}/contribute/hacker-guide.html">Hacker's Guide</a></li>
```

Comment:
well, I don't have a real strong opinion on this. I just thought it was a little weird that it was called "hacker's guide" everywhere but in one place.

hmm, how about sidestepping the issue by simply removing the link from the footer? I think we want new people to come in through the main Contribute page (aka "How to Help"), rather than through the Hacker's Guide.